cmd/kritis/admission/main.go

```diff
@@ -65,8 +56,37 @@ func main() {
 		fmt.Println(version.Commit)
 		return
 	}
+
+	kritisConfigs, err := kritisconfig.KritisConfigs("default")
+	if err != nil {
+		errMsg := fmt.Sprintf("error getting kritis config: %v", err)
+		glog.Errorf(errMsg)
+		return
+	}
+	if len(kritisConfigs) == 0 {
+		glog.Errorf("No KritisConfigs found in any namespace")
+		return
```

Comment:
If the user doesn't deploy a KritisConfig, should kritis assume defaults? Or is it required that the user also deploys a KritisConfig?

Reply:
Good point, I think it should. For now, there's no need to require that the user deploys a KritisConfig. Updated the logic to reflect this.